tensorflow/python/ops/array_ops.py

```diff
@@ -59,8 +59,19 @@
 @tf_export("identity")
 @dispatch.add_dispatch_support
 def identity(input, name=None):  # pylint: disable=redefined-builtin
-  r"""Return a tensor with the same shape and contents as input.
+  r"""Identity returns the value of the input tensor at the time of evaluation.
```

Comment:
Tensors have values which do not change over time so specifying the time is weird

Reply:
@alextp removed that line. So there is no change in that line. Thanks!